frontend/src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -24,7 +24,24 @@ export class DashboardComponent implements OnInit {
     }
     this.isSubmit = lectureData
   }
+  getVideoID(text) {
```

Comment:
Function name should accurately describe what it's trying to do. So, I believe getVideoLink or something like that would be more appropriate.